README.md

```diff
@@ -4,26 +4,26 @@
 
 ## What is Rafiki?
 
-Rafiki is open source software that allows an [Account Servicing Entity](https://rafiki.dev/concepts/account-servicing-entity/) to enable [Interledger](https://rafiki.dev/concepts/interledger-protocol/overview/) functionality on its users' accounts.
+Rafiki is an open source software that allows an [Account Servicing Entity](https://rafiki.dev/concepts/account-servicing-entity/) to enable [Interledger](https://rafiki.dev/concepts/interledger-protocol/overview/) functionality on its users' accounts.
```

Comment:
Can we please leave that without the "an"?

Reply:
I am sorry I am unaware how to leave out the an. Thank you.

Reply:
Just don't add the "an" here, please. "Rafiki is open source software ..." is grammatically correct.